content/docs/contributing/to-docs/style-guide/format.md

```diff
@@ -2,11 +2,11 @@
 title: Formatting O3DE Documentation
 description: A reference for all of the typesetting and formatting rules for the Open 3D Engine (O3DE) documentation.
 linktitle: Formatting
-weight: 200
+weight: 300
 toc: true
 ---
 
-The **Open 3D Engine (O3DE)** documentation is written in Markdown syntax. With Markdown, sometimes there are multiple methods to achieve the same result. For example, you can enclose words in underscores (`_`) or asterisks (`*`) to create italics. In these situations, it is best to use one method throughout the documentation. To keep both the documentation Markdown source files, and the O3DE documentation on-page presentation consistent, adhere to the following basic documentation standards.
+The **Open 3D Engine (O3DE)** documentation is written in **Markdown syntax**. With Markdown, sometimes there are multiple methods to achieve the same result. For example, you can enclose words in underscores (`_`) or asterisks (`*`) to create italics. In these situations, it is best to use one method throughout the documentation. To keep both the documentation Markdown source files, and the O3DE documentation on-page presentation consistent, adhere to the following basic documentation standards.
```

Comment:
We should probably mention we use **Goldmark Markdown** specifically. We mention it later on in this topic.